remove_missed.py

```diff
@@ -0,0 +1,63 @@
+import toolforge
+import argparse
+from constants import DATABASE_NAME
+import utils.db_access as db_acc
+
+
+def remove_missed_contents(user_db_port, user, password):
```

Comment:
Well, I kept it separate because its functionality is unique. I couldnt find another file to merge it with. 
This file gets run everytime API and DB fetching is completed.
Let me know where it fits best.

Reply:
Ah, it _does_ run every time. I'm sorry, I haven't noticed that. When it's ok to leave it here.